### PR TITLE
Python dependencies

### PR DESCRIPTION
**Pull request by EduPonz**
Signed-off-by: EduPonz <eduardoponz@eprosima.com>

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/248/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/250/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/251/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/252/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/253/)

File: README.md [hunk 1]
```diff
@@ -1,159 +1,97 @@
...
-    ```bash
-    $ git clone https://github.com/foonathan/memory.git
-    $ cd memory
-    $ git submodule update --init --recursive
-    $ mkdir build && cd build
-    $ cmake ..
-    $ cmake --build . --target install
-    ```
+1. The RST follows the style guidelines
+1. The HTML is build correctly
```
Comment (FranFin) [inline README.md:87]:
```suggestion
1. The HTML is built correctly
```

File: README.md [hunk 2]
```diff
@@ -1,159 +1,97 @@
...
-[colcon](https://colcon.readthedocs.io) is a command line tool to build sets of software packages.
-This section explains to use it to compile easily Fast-RTPS and its dependencies.
-First install ROS2 development tools (colcon and vcstool):
+To generate the documentation in a HTML format for a specific branch of Fast-RTPS run:
 
 ```bash
-pip install -U colcon-common-extensions vcstool
+cd ~/fastrtps-docs
+source fastrtps-docs-venv/bin/activate
+FASTRTPS_BRANCH=<branch> make html
```
Comment (IkerLuengo) [inline README.md:60]: This looks like the branch variable is mandatory.

I would start with a command without specifying the Fast-RTPS branch (i.e., using master, that will be most useful for users) and then explain how to use another branch different form master

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/258/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/259/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/260/)

File: README.md (hunk 2 above)
Comment (EduPonz) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (EduPonz) [reply README.md]: done

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/263/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/264/)

**Pull request merged**